src/interfaces.ts

```diff
@@ -326,10 +326,14 @@ export interface WatchedCronItems {
   release: () => void;
 }
 
+export type CronItemOptions = {
+  [key in keyof ParsedCronItemOptions]?: ParsedCronItemOptions[key];
+};
+
 /**
  * a.k.a. `opts`, this allows you to change the behaviour when scheduling a cron task.
  */
-export interface CronItemOptions {
+export interface ParsedCronItemOptions {
   /** How far back (in milliseconds) should we backfill jobs when worker starts? (Only backfills since when the identifier was first used.) */
   backfillPeriod: number;
```

Comment:
Instead, make the field optional:

```suggestion
export interface CronItemOptions {
  /** How far back (in milliseconds) should we backfill jobs when worker starts? (Only backfills since when the identifier was first used.) */
  backfillPeriod?: number;
```

and then later make it explicit just this one field is being changed internally:

```ts
export interface ParsedCronItemOptions extends CronItemOptions {
  backfillPeriod: number;
}
```

Reply:
This seems to be more inline with the existing style of using interfaces over types. I'll adjust it to this.